core/src/main/java/org/apache/accumulo/core/clientImpl/ImportConfigurationImpl.java

```diff
@@ -19,13 +19,15 @@
 package org.apache.accumulo.core.clientImpl;
 
 import org.apache.accumulo.core.client.admin.ImportConfiguration;
+import org.apache.accumulo.core.client.admin.TabletHostingGoal;
 
 import com.google.common.base.Preconditions;
 
 public class ImportConfigurationImpl implements ImportConfiguration, ImportConfiguration.Builder {
   private boolean built = false;
   private boolean keepOffline = false;
   private boolean keepMappingsFile = false;
+  private TabletHostingGoal initialHostingGoal = null;
```

Comment:
Would it simplify things if instead of null, a default (ONDEMAND ?) be set here?  It may eliminate the need to keep checking for null.

Reply:
Agreed. ONDEMAND is the default tablet hosting state when a table is created.

Reply:
Should be addressed now